todo_app/integration_test.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import dotenv
+from dotenv import load_dotenv,find_dotenv
+import os
+from os.path import join, dirname
+from dotenv import find_dotenv
+from todo_app.app import create_app
+from unittest.mock import patch,Mock
+#import responses
+import requests
+#import vcr
+
+
+@pytest.fixture
+def client():
+# Use our test integration config instead of the'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+# Create the new app.
+#test_app = app.create_app()
+    test_app = create_app()
+# Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+
+@patch('requests.get')
+def test_index_page(mock_get_requests, client):
+# Replace call to requests.get(url) with our own function
+     mock_get_requests.side_effect = mock_get_lists
+     test_board_id = '6052828032dafa5707bf5reg'
+     # url = f'http://localhost:5000/'
+     url = f'/'
+     response = client.get(url)
```

Comment:
We should make an assertion after this call to check that we're not getting any issues - just because the test hasn't errored doesn't mean it's succeeded!
E.g. here we could verify that some of your fake responses are being displayed on the page, a simple example might be `assert "test data" in response.data.decode()`